Actions: How to run jobs on issue_comment only if PR (#438)
github/docs: content/actions/reference/events-that-trigger-workflows.md

    types: [created, deleted]
```

> The `issue_comment` event occurs for both, comments on issues as well as pull requests. To run jobs only in one of the cases, you can check the event payload for the `issue.pull_request` attribute and use it as condition to skip jobs.

Author: Nice, just one little change I would suggest:

> To ~~check~~ **determine** whether the `issue_comment` event was triggered from an issue or pull request, you can **check** the event payload

> The `issue_comment` event occurs for both, comments on issues as well as pull requests. To run jobs only in one of the cases, you can check the event payload for the `issue.pull_request` attribute and use it as condition to skip jobs.

Author: ```suggestion
The `issue_comment` event occurs for comments on both issues and pull requests. To determine whether the `issue_comment` event was triggered from an issue or pull request, you can check the event payload for the `issue.pull_request` property and use it as a condition to skip a job.

For example, you can choose to run the `pr_commented` job when comments are added to a pull request or the `issue_commented` job when comments are added to an issue. 
```

```yaml
on: issue_comment

jobs:

>   pr_commented:

Reviewer: Let's add a comment for readers that skim the workflow file!

```suggestion
  // This job only runs when a pull request comment is created
  pr_commented:
```

Reviewer 2: Think the comments need to be with `#` because it's in a Yaml file

Author: `# Comment` here as well

Reviewer: Good catch @nschonni @Simran-B 🙈 Yes definitely should be `#`.

>     if: ${{ github.event.issue.pull_request }}

Reviewer: What do you think about adding the opposite case too?

```suggestion
    name: New PR comment
    if: ${{ github.event.issue.pull_request }}
    runs-on: ubuntu-latest
  
    steps:
      - run: echo "New PR comment"
  
  // This job only runs when an issue comment is created
  issue-commented:
    name: New issue comment
    if: ${{ !github.event.issue.pull_request }}
    runs-on: ubuntu-latest
  
    steps:
      -  run: echo "New issue comment"
```

Author: Yeah, why not.

The comment should start with `#` not `//`

Author: This job only runs when an issue **comment is created** - this isn't actually true for the example. It will trigger for `created`, `edited` and `deleted` (default). I changed the comment to avoid activity types altogether in this example.

Reviewer: Good catch @Simran-B! 🙌

    types: [created, deleted]
```

The `issue_comment` event occurs for comments on both issues and pull requests. To determine whether the `issue_comment` event was triggered from an issue or pull request, you can check the event payload for the `issue.pull_request` property and use it as a condition to skip a job.

> For example, you can choose to run the `pr_commented` job when comments are added to a pull request, and the `issue_commented` job when comments are added to an issue.

Reviewer: @Simran-B sorry for the back and forth! Based on your observation in https://github.com/github/docs/pull/438/files#r517500086, I think we should align this comment too!

```suggestion
For example, you can choose to run the `pr_commented` job when comments events occur in a pull request, and the `issue_commented` job when comments events occur in an issue.
```

Author: @rachmari Good catch! And no worries, I appreciate your feedback very much.

```suggestion
For example, you can choose to run the `pr_commented` job when comment events occur in a pull request, and the `issue_commented` job when comment events occur in an issue.
```
